commands/utils/consts.go

```diff
@@ -33,7 +33,7 @@ const (
 	JFrogUrlEnv            = "JF_URL"
 	jfrogXrayUrlEnv        = "JF_XRAY_URL"
 	jfrogArtifactoryUrlEnv = "JF_ARTIFACTORY_URL"
-	jfrogRemoteRepo        = "JF_FROGBOT_REPO"
+	jfrogReleasesRepo      = "JF_RELEASES_REPO"
```

Comment:
To be compatible with the other varoables defined here - 
jfrogReleasesRepo --> jfrogReleasesRepoEnv

Reply:
done